### PR TITLE
added blog 2

### PR DESCRIPTION
**Pull request by abhijeetmanhas**
I have added second blog titled "in a parallel universe" which was already published on STAC medium. 
@shreyasbapat please review it.
Screenshot (62)

### CODE REVIEW

**Comment by shreyasbapat** [comment]
Why are 37 file changes?

File: _site/CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+blog.stac-iitmandi.org
```
Comment (shreyasbapat) [inline _site/CNAME]: CNAME is already added

File: _site/COPYING [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Copyright (c) 2019 STAC Web Development Team, WowThemes.net
```
Comment (shreyasbapat) [inline _site/COPYING]: No need to add this either

File: _site/about.html [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+
+<!-- Begin Jekyll SEO tag v2.5.0 -->
+<title>Space Technology and Astronomy Cell | AstroThoughts</title>
+<meta name="generator" content="Jekyll v3.8.5" />
+<meta property="og:title" content="Space Technology and Astronomy Cell" />
+<meta property="og:locale" content="en_US" />
+<meta name="description" content="Blog of the Space Technology and Astronomy Cell" />
+<meta property="og:description" content="Blog of the Space Technology and Astronomy Cell" />
+<link rel="canonical" href="http://localhost:4000/about" />
+<meta property="og:url" content="http://localhost:4000/about" />
```
Comment (shreyasbapat) [inline _site/about.html]: why localhost

**Review by shreyasbapat — CHANGES_REQUESTED** [review]
Some questions, though I don't get to understand why there are so many changes

**Comment by abhijeetmanhas** [comment]
I found most of the changes are in _site folder, which is formed when you build or compile your blog, _site and cache should be put in .gitignore file which didn't pre-existed, I will do this know.

**Comment by abhijeetmanhas** [comment]
> Why are 37 file changes?

Now only 4. I have added .gitignore.

File: _posts/2018-07-08-In-a-Parallel-Universe.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+George Ellis
+
+Neil Turok and more…
+
+But Physics is science, not philosophy. Hopefully, someday, one of us will design, build and conduct an experiment to test these hypotheses. We shall overcome this debate, someday.
+
+Maybe you believe one of the above models, maybe none. Screw it. How does it matter, we are all living inside the Matrix. Or maybe, not living, either.
+
+Written by: Anant Mishra
```
Comment (shreyasbapat) [inline _posts/2018-07-08-In-a-Parallel-Universe.md]: Remove this line

**Pull request merged**